### DISCUSSION TITLE
How to increase the chats fontsize ?

### DISCUSSION THREAD

**Discussion by wallygit** [eneshecan/whatsapp-for-linux#206] [Q&A] [answered]
How to increase the chats fontsize ?
Any accessibility features ?
I'm using WhatsApp for Linux on Kubuntu 20.0.4
regards wally

**Comment by xeco23** [comment, marked as answer]
Sure, it would be possible to implement a `Font Size` setting if the zoom in/out feature doesn't work for you. Please create an issue so that we can keep track of it.
